HexCtrl/src/CHexCtrl.cpp

```diff
@@ -772,6 +772,15 @@ auto CHexCtrl::GetSelection()const->std::vector<HEXSPAN>
 	return m_pSelection->GetData();
 }
 
+wchar_t CHexCtrl::GetUnprintableCharacter()const
+{
+	assert(IsCreated());
+	if (!IsCreated())
+		return { L' ' };

```

Comment:
Returning a space is not logically correct here, just an empty `{ }` char is.

Reply:
Fixed